src/app/lib/styledGrid.js

```diff
@@ -1,13 +1,10 @@
 import styled from 'styled-components';
-import { C_OAT_LHT } from '@bbc/psammead-styles/colours';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
 import { layoutGridWrapper, layoutGridItemConstrained } from './layoutGrid';
 
 export const Wrapper = styled.div`
```

Comment:
Could you please call this `GhostWrapper` so that when using it, it's clear that it is ghost-coloured.